osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderPlacementBlueprint.cs

```diff
@@ -230,7 +230,7 @@ private bool canPlaceNewControlPoint([CanBeNull] out PathControlPoint lastPoint)
 
         private void updateSlider()
         {
-            HitObject.Path.ExpectedDistance.Value = snapProvider?.FindSnappedDistance(HitObject, (float)HitObject.Path.CalculatedDistance) ?? (float)HitObject.Path.CalculatedDistance;
+            HitObject.Path.ExpectedDistance.Value = HitObject.DifficultyControlPoint.SliderVelocity * (snapProvider?.FindSnappedDistance(HitObject, (float)HitObject.Path.CalculatedDistance) ?? (float)HitObject.Path.CalculatedDistance);
```

Comment:
This can be simplified to `SliderPathExtensions.SnapTo(...)`, the formula is the same.